### PR TITLE
Unsaved changes primary button label changed to Discard

### PR DESCRIPTION
**Pull request by sandikbarr**
#325 "Discard Changes" doesn't feel right. It's too big, and all of our other buttons are a single verb.
screen shot 2017-02-02 at 1 20 03 pm

### CODE REVIEW

File: src/app/upload/shared/audio-cancel.directive.ts [hunk 1]
```diff
@@ -19,10 +19,10 @@ export class AudioCancelDirective {
     if (this.publishAudioCancel.isUploading) {
       this.cancelAndDestroy();
     } else {
-      this.modal.prompt(
-        'Really delete audio file?',
-        'This action cannot be undone.',
```
Comment (sandikbarr) [inline src/app/upload/shared/audio-cancel.directive.ts]: this seemed like a lie because discard brings the file back

**Comment by sandikbarr** [comment]
ok! let me know what you think of this refactor @gcampo88

**Comment by gcampo88** [comment]
missed one!: https://github.com/PRX/publish.prx.org/blob/master/src/app/series/series.component.ts#L88

:)

**Comment by sandikbarr** [comment]
oops, fixed!

**Comment by gcampo88** [comment]
👍

**Pull request merged**